File: app/src/main/java/fr/niavlys/dev/trajets/config/AppConfig.java
package fr.niavlys.dev.trajets.config;

import java.util.ArrayList;
import java.util.List;

public class AppConfig {
    private final List<ConfigItem> transports;
    private final List<ConfigItem> stations;
    private final List<ContactItem> contacts;

    public AppConfig() {
        transports = new ArrayList<>();
        stations = new ArrayList<>();
        contacts = new ArrayList<>();
    }

    public List<ConfigItem> getTransports() {
        return transports;
    }

    public List<ConfigItem> getStations() {
        return stations;
    }

    public List<ContactItem> getContacts() {
        return contacts;
    }
}

File: app/src/main/java/fr/niavlys/dev/trajets/config/AppConfig.kt
package fr.niavlys.dev.trajets.config

data class AppConfig(
    val transportCategories: MutableList<CategoryItem> = mutableListOf(),
    val transports: MutableList<ConfigItem> = mutableListOf(),
    val stations: MutableList<ConfigItem> = mutableListOf(),
    val contacts: MutableList<ContactItem> = mutableListOf(),
    val contactGroups: MutableList<ContactGroupItem> = mutableListOf(),
    val messagePresets: MutableList<PresetItem> = mutableListOf()
)
